test/elixir_internal_certificate/accounts_test.exs

```diff
@@ -0,0 +1,172 @@
+defmodule ElixirInternalCertificate.AccountsTest do
+  use ElixirInternalCertificate.DataCase
+
+  alias ElixirInternalCertificate.Accounts
+  alias ElixirInternalCertificate.Accounts.{User, UserToken}
+
+  setup do
+    %{user: insert(:user)}
```

Comment:
We don't prefer to use the hook to set up the test data @nvminhtue as it's hard to customize the test data later, we prefer to set the test data in each test to make sure the tests are isolated

https://nimblehq.co/compass/development/testing/#general-best-practices

![image](https://user-images.githubusercontent.com/11751745/172758584-dbc15921-e9db-44a9-a3bd-2b3926a2c3c1.png)